src/gui/widgets/multi_page.cpp

```diff
@@ -154,7 +169,8 @@ void swap_grid(grid* g,
 void multi_page::finalize(const std::vector<string_map>& page_data)
 {
 	assert(generator_);
-	generator_->create_items(-1, page_builder_, page_data, nullptr);
+	//FIXME
+	generator_->create_items(-1, page_builders_.begin()->second, page_data, nullptr);
```

Comment:
the issue here is that page_data currently doesn't support specifying the page type.